src/result.rs

```diff
@@ -248,3 +278,34 @@ impl CqlValueWrapper {
         }
     }
 }
+
+fn map_column_type_to_cql_value(typ: &ColumnType) -> CqlType {
```

Comment:
The name is very incorrect. `CqlValue` is an enum that can contain an arbitrary CQL value.

Reply:
```suggestion
fn map_column_type_to_cql_type(typ: &ColumnType) -> CqlType {
```